src/dodal/devices/shutter.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class OpenState(str, Enum):
+    CLOSED = "closed"
+    OPEN = "open"
+    OPENING = "opening"
+    CLOSING = "closing"
+    FAULT = "fault"
+
+
+class Shutter(StandardReadable, Movable):
+    def __init__(self, prefix: str, name: str):
+        self.position = epics_signal_rw(
+            write_pv=prefix + "CTRL2",
+            read_pv=prefix + "STA",
+            datatype=OpenState,
+            name=name,
```

Comment:
shutter.position.name shouldn't be equal to shutter.name (as this change currently does). shutter.position.name should retain the default "" (i.e., don't pass it into this call) and it will be named by the DeviceCollector.

Reply:
ok